arrays/victor/C#_soft-188/4_task/4_task/4_task/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Linq;
+
+namespace _4_task
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int[] mas = { 1, 2, 5, 78, 2, 34, 248, 11, 9, 29, 0, 270 };
+            Console.WriteLine("Масив = 1, 2, 5, 78, 2, 34, 248, 11, 9, 29, 0, 270");
+            int minValue = mas.Min();
```

Comment:
In this task you can't use C# methods. Try to find the min element without using Min() method.